### PR TITLE
Fetch pre-compiled rebar3 instead of building it locally

### PR DESCRIPTION
**Pull request by kivra-pauoli**
This is a partial fix for #15, as well as (hopefully) issues with e.g. `escript` not being available at compile-time, as described in https://github.com/asdf-vm/actions/issues/585.

# Advantages

* faster build time (no build, actually 😄)
* wider Erlang version range - compatibility (since it's build for Erlang <cur-2> by [erlang/rebar3](https://github.com/erlang/rebar3)): "The 3 newest major Erlang releases are supported at any given time: if the newest version is OTP-27, building with versions as old as OTP-25 will be supported, and produce an executable that will work with those that follow."

### CODE REVIEW

File: bin/install
Comment (kivra-pauoli) [inline bin/install:1]: Probably best seen with `Hide whitespace` enabled as part of this included changes in formatting.

**Comment by kivra-pauoli** [comment]
I'll also update CI to use `asdf-vm/actions` and thus replicate a real user situation (`erlang` installed via `asdf` plugin)...

**Comment by Stratus3D** [comment]
Thanks for the PR @kivra-pauoli ! Looks good to me. I'm going to do some testing and then merge later.

**Comment by Stratus3D** [comment]
Tested locally and seems good to me. And much faster!

**Pull request merged**